site/docs/types.md

```diff
@@ -5,71 +5,65 @@ title: Parameter Types
 permalink: /docs/types.html
 ---
 
-Reference documentation for common parameter **types** expected by Vega specification properties.
+Reference documentation for common parameter **types** expected by Vega-Lite specification properties.
 
 {:#reference}
 ## Parameter Type Reference
 
-- [Any](#Any)
-- [Array](#Array)
-- [Boolean](#Boolean)
-- [Color](#Color)
-- [Number](#Number)
-- [Object](#Object)
-- [String](#String)
-- [URL](#URL)
-- [DateTime](#DateTime)
-- [Field](#Field)
-- [Signal](#Signal)
-- [Compare](#Compare)
-- [Expr](#Expr)
-- [Value](#Value)
-- [ColorValue](#ColorValue)
-- [FieldValue](#FieldValue)
+- [Any](#any)
+- [Array](#array)
+- [Boolean](#boolean)
+- [Color](#color)
+- [Number](#number)
+- [Object](#object)
+- [String](#string)
+- [URL](#url)
+- [DateTime](#datetime)
+- [Expressions](#expression)
 {: .column-list }
 
 ## Literal Values
 
-<a name="*"></a><a name="Any" href="#Any">#</a>
+<a name="*"></a><a name="any" href="#any">#</a>
```

Comment:
Please search and make sure we don't refer to these anchor names with Titlecase from other pages.

Reply:
I searched for `types.html`.